themes/custom/ts_wrin/js/components/ts_header_nav.js

```diff
@@ -152,13 +152,37 @@ export default function(context) {
       }
     }
 
+    function stickyScrollMobile() {
+      const mobileStickyNav = document.querySelector(
+        ".page-node-type-simple-page .layout__region--menu"
+      );
+      const mobileStickyParent = document.querySelector(
+        ".page-node-type-simple-page .simple-page__title"
+      );
+
+      if (mobileStickyNav) {
```

Comment:
There's another version of the sticky nav needed for the About page, which is a view:
https://pr-1254-wriflagship.pantheonsite.io/about/experts-staff

This works though:
https://pr-1254-wriflagship.pantheonsite.io/about/mission-values

Reply:
Ah, got it. Also fixed a resize bug while I was in there.